### ISSUE TITLE
Explorer: when browsing bucket based remotes, the download URL is wrong

### ISSUE THREAD

**Issue by ncw** [negative0/rclone-webui-react#11]
If you can make sure the download URL for Bucket based remotes is
```
http://127.0.0.1:5572/[memstore:10000files]/zebafep1lod/behoxo
```
Instead of 
```
http://127.0.0.1:5572/[memstore:10000files:zebafep1lod]/behoxo 
```
Then it will work fine.

Note there is a bug in rclone here too which I fixed here: https://github.com/ncw/rclone/commit/f681d3299690422589b305cfd56582d5fa4bcdb4

**Comment by negative0** [comment]
Fixed in commit [https://github.com/negative0/rclone-webui-react/commit/b5cd3fe0d85591ae7650407cb609afe57c01cb2b](here)

**Comment by ncw** [comment]
:-)  I don't think you've pushed that one yet?

I normally just write "Fixes #11` in the commit message and let the push to the repo close the issue.

**Comment by negative0** [comment]
I have pushed that one. The link was wrong. I added # but I had referenced an old issue
